core/InstArchInfo.cpp

```diff
@@ -50,11 +50,12 @@ namespace olympia
     };
 
     const InstArchInfo::UopGenMap InstArchInfo::uop_gen_type_map = {
-        {"ARITH",               InstArchInfo::UopGenType::ARITH},
-        {"ARITH_SINGLE_DEST",   InstArchInfo::UopGenType::ARITH_SINGLE_DEST},
-        {"ARITH_WIDE_DEST",     InstArchInfo::UopGenType::ARITH_WIDE_DEST},
-        {"ARITH_MAC",           InstArchInfo::UopGenType::ARITH_MAC},
-        {"ARITH_MAC_WIDE_DEST", InstArchInfo::UopGenType::ARITH_MAC_WIDE_DEST},
+        {"ELEMENTWISE",               InstArchInfo::UopGenType::ELEMENTWISE},
```

Comment:
Fix formatting.

Reply:
Should I use the prior convention or the formatting by the top level .clang-format?

Reply:
Have it match whatever clang-format does.